ldm/invoke/globals.py

```diff
@@ -0,0 +1,20 @@
+'''
+ldm.invoke.globals defines a small number of global variables that would
```

Comment:
We may want to revisit this in the node rewrite. Globals may not make sense in some configurations.

Reply:
I reluctantly used one global here to avoid adding a root directory argument to multiple nested functions. Agree it should go away in the node rewrite (along with most or all of the ldm.generate and ldm.invoke.args code)